README.MD

```diff
@@ -74,6 +74,74 @@ The loaders accepts the name of a loader that should be applied in advance. E.g.
 }
 ```
 
+### Custom HTML Translation extractors
+
+The htmlLoader supports registering custom HTML text extractors and rewrites the AngularJS extractor to be based on the same API.
```

Comment:
I find this sentence a little confusing. Plus is the _AngularJS extractor_ rewritten, or an extra one is added.